src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/StoreV2/ContactDiaryStoreV2.swift

```diff
@@ -7,18 +7,9 @@ import UIKit
 import FMDB
 import OpenCombine
 
-protocol DateProviding {
-	var today: Date { get }
-}
-
-struct DateProvider: DateProviding {
-	var today: Date {
-		Date()
-	}
-}
 
 // swiftlint:disable:next type_body_length
-class ContactDiaryStore: DiaryStoring, DiaryProviding {
+class ContactDiaryStoreV2: DiaryStoringV2, DiaryProvidingV2 {
```

Comment:
I'm just writing this here as a general question: Why do we store the risk levels per day in the contact diary store at all? Aren't they contained in the normal riskCalculationResult? Isn't this just redundant data?

Reply:
RiskCalculationResult did not store that data before it was only a step inside the calculation. But the main issue is that we need this information inside the journal on a daily basis.
The tec-spec suggested to store these information as part of the contact journal and automatically delete after 15 days with the rest.

Reply:
But we do store it now, don't we? I don't see why we should keep and update it redundantly if it's now saved by the risk calculation.
I can't find anything in the tech spec that suggests storing this data with the contact journal, can you help me out?